server/chat-commands/moderation.ts

```diff
@@ -738,11 +739,10 @@ export const commands: ChatCommands = {
 		const lockMessage = `${name} was locked from talking${durationMsg} by ${user.name}.` + (userReason ? ` (${userReason})` : "");
 		this.addModAction(lockMessage);
 		// Notify staff room when a user is locked outside of it.
-		if (room.roomid !== 'staff' && Rooms.get('staff')) {
-			Rooms.get('staff')?.addByUser(user, `<<${room.roomid}>> ${lockMessage}`);
+		if (!room || room.roomid !== 'staff') {
+			 Rooms.get('staff')?.addByUser(user, `${room ? `<<${room.roomid}>>` : `<PMs: ${this.pmTarget}>`} ${lockMessage}`);
```

Comment:
```suggestion
			Rooms.get('staff')?.addByUser(user, `${room ? `<<${room.roomid}>>` : `<PM:${this.pmTarget}>`} ${lockMessage}`);
```

I'm really confused why the linter isn't complaining about this.